interact.js

```diff
@@ -0,0 +1,122 @@
+
+//이메일 input에서 focus out 할 때, 값이 없을 경우 아래에 “이메일을 입력해 주세요.”
+//이메일 input에서 focus out 할 때, 이메일 형식에 맞지 않는 값이 있는 경우 아래에 “올바른 이메일 주소가 아닙니다.”
+
+const inputel=document.querySelector('#emailinput'); //이메일 input 박스
+const emailframe=document.querySelector('#emailframe'); //이메일 input 박스 frame
+const errmsg=document.createElement('p'); //에러메시지
+let emailtest = /^[^\s@]+@[^\s@]+\.[^\s@]+$/; //이메일 테스팅 코드
+
+function emaininput(e){
+  if(e.target.value==''){
+    errmsg.textContent='이메일을 입력해 주세요.';
+    emailframe.append(errmsg);
+    console.log(1);
+    console.log(e.target.type);
+  }
+  else{
+    if(!emailtest.test(e.target.value)){ //이메일 확인 테스트
+      errmsg.textContent='올바른 이메일 주소가 아닙니다.';
+      emailframe.append(errmsg);
+    }
+    else{
+      if(errmsg.textContent){ //만약에 올바른 이메일 주소를 작성하였으면 에러메시지 삭제
+        errmsg.remove();
+      }
+      console.log(2);
+    }
+    
+  }
+}
+
+inputel.addEventListener('focusout',emaininput);
+
+
+//이메일: test@codeit.com, 비밀번호: codeit101 으로 로그인 시도할 경우, “/folder” 페이지로 이동합니다.
+
+const btnlogin=document.querySelector('.login');
+const inputel2=document.querySelector('#passwordinput');
+
+function clicklogin(){
+  if(inputel.value=='test@codeit.com' && inputel2.value=='codeit101'){
+    location.href='/folder.html';
+  }
+}
+
+btnlogin.addEventListener('click',clicklogin);
```

Comment:
r: 폼 정보 전송을 위한 [submit](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event) 이벤트에 대해 찾아보시고 로그인 버튼 클릭 대신 폼 제출 이벤트를 이용해 로그인 요청을 구현하시는 것을 추천드려요.

`submit` 이벤트는 폼에 포커스가 된 상태에서 `enter` 키를 입력받아도 발생되는 웹 표준 폼 API 중 하나입니다.

```suggestion
loginForm.addEventListener('submit', login);
```